content/level-defaults/de.yaml

```diff
@@ -430,10 +430,10 @@
 
 13:
     start_code: |-
-        name = ask 'what is your name?'
-        leeftijd = ask 'what is your age?'
-        if name is 'Hedy' and age is 2
-            print 'You are the real Hedy!'
+        Name = ask 'Wie heißt Du?'
+        Alter = ask 'Wie alt bist Du?'
+        if Name is 'Hedy' and Alter is 2
+            print 'Du bist die echte Hedy!'
```

Comment:
Is Hedy actually female..? @Felienne?

Reply:
Yes!! Named after Hedy Lamarr!

Reply:
Okay. Always assumed it due to how the name sounds, but I wasn't entirely sure and wanted to confirm.